bHY6yh-de4030/cod-yyyiou_bHY6yh-de4030.xml

```diff
@@ -0,0 +1,259 @@
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e396">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e445">Idem 26</ref> <quote xml:id="bHY6yh-de4030-Qd1e398">Vi<lb ed="#P" break="no"/>gilate 
+              et orate ut non interetis in <sic>tempta<lb ed="#P" break="no"/>tationem</sic> 
+              spiritus quidem promptus est caro autem infirma</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e406">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e459">Marcus in euangelio capitulo XI</ref> <quote xml:id="bHY6yh-de4030-Qd1e410">omnia
+              <lb ed="#C"/>quaecumque orantes petitis credite quia
+              <lb ed="#C"/>accipietis et ueniet uobis<!--manuscript makes the rest of the passage a new paragraph--> ¶ Idem cum stabitis
+              <lb ed="#C"/>ad orandum dimittite si quid habetis aduer<lb ed="#P" break="no"/>sus 
+              aliquem ut pater uester qui incelis
+              <lb ed="#C"/>dimittat uobis peccata uestra</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e422">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e477">Lucas XI capitulo</ref> <quote xml:id="bHY6yh-de4030-Qd1e426">Cum
+              <lb ed="#C"/>oratis dicite pater noster quie est in caelis etcetera
+              <lb ed="#C"/>et dicite etcetera</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e441">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e496">Idem 22</ref> <quote xml:id="bHY6yh-de4030-Qd1e443">orate ne intretis 
+              <lb ed="#C"/>intemptationem</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e449">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e508">Ioannes in euangelio capitulo IX</ref> <quote xml:id="bHY6yh-de4030-Qd1e453">peccatores
+              <lb ed="#C"/>deus non audit sed siquis dei cultor est
+              <lb ed="#C"/>et eius uoluntatem facit hunc exaudiet</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e461">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e522">Paulus I corinthiorum 
+              <lb ed="#C"/>Ie</ref> <quote xml:id="bHY6yh-de4030-Qd1e465">Si orem spiritus meus et lingua non oret
+              <lb ed="#C"/>mens mea sine fructum est</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e473">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e534">Idem ad collosenses
+              <lb ed="#C"/>capitulo III et penultimo</ref> <quote xml:id="bHY6yh-de4030-Qd1e475">Domine quodcumque facitis in nomine
+              <lb ed="#C"/>deum mihi iesu christi facite</quote>
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e489">
+              ¶ <ref xml:id="bHY6yh-de4030-Rd1e554">Cato capitulo fidelis</ref> <quote xml:id="bHY6yh-de4030-Qd1e493">Christe
+              <lb ed="#C"/>deum placauit illum ut crescicat oratio</quote>
+            </p>
+              <pb ed="#C" n="7-r"/>
+              <cb ed="#C" n="a"/>
+              <lb ed="#C"/>
+            <p xml:id="bHY6yh-de4030-d1e499">
```

Comment:
Page, column, and line breaks always need to be inside a `<p>` tag. See my change below.
```suggestion
            <p xml:id="bHY6yh-de4030-d1e499">
                <pb ed="#C" n="7-r"/>
                <cb ed="#C" n="a"/>
                <lb ed="#C"/>

```